sky/clouds/service_catalog/data_fetchers/fetch_fluidstack.py

```diff
@@ -19,6 +18,8 @@
 
 GPU_MAP = {
     'H100_PCIE_80GB': 'H100',
+    'H100_NVLINK_80GB': 'H100',
+    'A100_NVLINK_80GB': 'A100-80GB',
```

Comment:
Nice! It seems I getting the following "failed to map" list, do we want to add them as well?
```
Could not map QUADRO_RTX_5000_16GB
Could not map RTX_A4500_16GB
Could not map QUADRO_RTX_4000_12GB
Could not map RTX_A6000_24GB
Could not map Quadro_P5000_12GB
Could not map QUADRO_P5000_16GB
```

Reply:
Not really. I will suppress these.